### ISSUE TITLE
Not working anymore...

### ISSUE THREAD

**Issue by Greenpassion** [diegocr/CleanLinks#153]
Firefox 48.0.2 / **64** / **Multitask**
Not working anymore...
Is it the 64 version or the multitask enabled?
Thanks

**Comment by alekksander** [comment]
no e10 here, but extension doesn't work on 48.0.2

**Comment by geokis** [comment]
No idea what you are talking about...

Firefox **48.0.2 x64**
It is working at its best !

**Comment by alekksander** [comment]
sometimes it works… for example it's not possible to log in to aliexpress without adding it to the white list… but when i go to link from google, first it gives me:
https://www.google.pl/url?sa=t&rct=j&q=&esrc=s&source=web&cd=13&ved=0ahUKEwiNyfqTipHPAhUJtxQKHVG6CvsQFghPMAw&url=https%3A%2F%2Fpl.wikipedia.org%2Fwiki%2FGrafika&usg=AFQjCNFmav2s9SazINQyZYOYlr2sVfNJmA&sig2=No-H5QAa6UY9rk_A6uKfYg&cad=rja
and then redirects to:
https://pl.wikipedia.org/wiki/Grafika
Needless to say, links do not turn yellow (default highlight style [background:rgba(252,252,0,0.6); color: #000])

**Comment by Greenpassion** [comment]
Still not working...
Removed and installed again...no results...
Firefox 48.0.2 / 64 / **MULTITASK**

**Comment by alekksander** [comment]
same issue on nightly with separate profile

**Comment by geokis** [comment]
Come on guys...
Deactivate the experimental **multiPROCESS** option and the problem is gone...

See also  #120

**Comment by alekksander** [comment]
please read carefully!
i do not use multiprocess! that is: Multiprocess Windows    0/1 (Disabled)

**Comment by alekksander** [comment]
okay so i re-added button to the toolbar… and it worked again! removing the icon disables this extension.
Filled out issue:
https://github.com/diegocr/CleanLinks/issues/154

**Comment by Greenpassion** [comment]
Ok!
So I disabled the multi**PROCESS** thing and it went back to normality...
Thanks a lot!

**Comment by Korpip** [comment]
Adding the icon allowed the addon to work for me as well on Firefox 49.0.1.

**Comment by jawz101** [comment]
@geokis multiprocess isn't experimental anymore.  It's in the stable release of Firefox.  The fact it's being staged is because many add-on developers haven't made their add-ons multiprocess aware.

**Comment by diegocr** [comment]
fwiw, in August i did started porting CL to be fully e10s compatible to an state that it's currently functional, however once it was made compatible i did realized an unfortunate fact, Firefox is unable to properly unload frame-scripts and any modules loaded through them, turning re-startless extensions into requiring a restart between upgrades, which is quite odd.

**Comment by GitCurious** [comment]
Does that mean the e10s compatible build actually works - but it just means that a restart is needed to upgrade to a new version ?

Can we voluntarily "Beta Test" the working version anyway ? 

(I have never understood the necessity or desire for restartless addons, I always restart after installing anything new anyway)

**Comment by diegocr** [comment]
Yes, that is what it means.

I can push what i've done till now, but first i need to clean up the code a bit, since is full of debugging stuff i've added while trying to deal with that.

**Comment by piejanssens** [comment]
What's up with e10s support, having to restart is really no issue comparing to not being compatible at all..

**Comment by MikeZeDev** [comment]
Any news on this? Having to restart is not a major problem and, maybe i am wrong but, not related to e10s validation

**Comment by Quantum00** [comment]
I agree with piejanssens and MikeZeDev. Even if it needs a restart we still need it.

**Comment by alekksander** [comment]
yes, please fix, it's getting urgent.

**Comment by copywrong** [comment]
Still waiting.

**Comment by jawz101** [comment]
Fwiw if it has to be e10s people can use Pure URL add-on.  I think it does the same thing as this one

**Comment by alekksander** [comment]
unfortunately it does not.
